server/drivers/hd44780.c

```diff
@@ -157,6 +157,7 @@ HD44780_init(Driver *drvthis)
 	p->line_address 	= drvthis->config_get_int(drvthis->name, "lineaddress", 0, LADDR);
 	p->have_keypad		= drvthis->config_get_bool(drvthis->name, "keypad", 0, 0);
 	p->have_backlight	= drvthis->config_get_bool(drvthis->name, "backlight", 0, 0);
+	p->isVFDDisplay		= drvthis->config_get_bool(drvthis->name, "VFDDisplay", 0, 0);
 	p->have_output		= drvthis->config_get_bool(drvthis->name, "outputport", 0, 0);
```

Comment:
Is the register set really generic to all VFD displays or just an extension of the PT6314  only. If it is the latter, please use a less generic name for both the configuration option and the variable. Ie something containing "pt6314".